### PR TITLE
[tempo-distributed] Include PodDisruptionBudget for the Ingester

### PR DESCRIPTION
**Pull request by zalegrala**
This change amends the podManagementPolicy that was introduced in #1057 to include a PodDisruptionBudget.  This will help ensure that the Ingesters won't take an outage during a deployment.

### CODE REVIEW

**Comment by osg-grafana** [comment]
cc @knylander-grafana

**Review by knylander-grafana — COMMENTED** [review]
Thank you for the pointer, Ursula.

**Comment by joe-elliott** [comment]
I think we do this on purpose for quick scale ups.

Wouldn't a "PodDisruptionBudget" cover what you're concerned about?

**Comment by zalegrala** [comment]
I think you are right @joe-elliott, this was pointed out by a few folks but I have't had time to test.  We probably need the PDB in our jsonnet also.

**Comment by zalegrala** [comment]
I've made an update to include the PDB.  Testing locally yields the following.
```
❯ k describe pdb chart-tempo-distributed-ingester
Name:             chart-tempo-distributed-ingester
Namespace:        default
Max unavailable:  1
Selector:         app.kubernetes.io/component=ingester,app.kubernetes.io/instance=chart,app.kubernetes.io/name=tempo-distributed
Status:
    Allowed disruptions:  1
    Current:              3
    Desired:              2
    Total:                3
Events:
  Type    Reason  Age                From               Message
  ----    ------  ----               ----               -------
  Normal  NoPods  37m (x2 over 37m)  controllermanager  No matching pods found
```

File: charts/tempo-distributed/templates/ingester/pdb.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{{- if .Values.podDisruptionBudget -}}
+apiVersion: policy/v1beta1
```
Comment (joe-elliott) [inline charts/tempo-distributed/templates/ingester/pdb.yaml:2]: `policy/v1` ?

**Review by joe-elliott — APPROVED** [review]

**Pull request merged**